### ISSUE TITLE
his modificaiton missing data

### ISSUE THREAD

**Issue by csbrasnett** [marrink-lab/vermouth-martinize#644]
running eg. `martinize2 -f clean.pdb -x cg.pdb -o topol.top -modify HIS:HIS-HP` raises an inconsistent data warning about atoms being covered by multiple blocks. Doesn't seem to affect other modifications. Can be fixed by changing the resnames in the input file, but that's not an ideal solution. 

Also to note, the HIS-HD modification seems to be wrong, the SC3 bead should become a TN6d, not a as currently is

**Comment by fgrunewald** [comment]
@csbrasnett could you provide some more information? Are yous sure the modification is detected?

**Comment by fgrunewald** [comment]
ohhhhh I think it has something to do with providing it using modify; if you were to just provide the correct pdb file it should work

**Comment by WangLiguo-kyrie** [comment]
Hi, this the the warning message 
'''WARNING - inconsistent-data - These atoms are covered by multiple blocks. This is a bad idea: {'274A-HIS17:HA', '265A-HIS17:C', '279A-HIS17:HE1', '267A-HIS17:CB', '269A-HIS17:ND1', '273A-HIS17:HN', '264A-HIS17:CA', '275A-HIS17:HB2', '276A-HIS17:HB1', '278A-HIS17:HD2', '268A-HIS17:CG', '271A-HIS17:CE1', '272A-HIS17:NE2', '263A-HIS17:N', '266A-HIS17:O', '270A-HIS17:CD2'}. This probably means the following output particles are wrong: {'A-HIS17:SC3', 'A-HIS17:BB', 'A-HIS18:SC1', 'A-HIS17:SC1', 'A-HIS18:SC3', 'A-HIS18:BB', 'A-HIS18:SC2', 'A-HIS17:SC2'}.'''

When we run the command listed above by Chris.

**Comment by WangLiguo-kyrie** [comment]
> ohhhhh I think it has something to do with providing it using modify; if you were to just provide the correct pdb file it should work. Or you can use the -mutate flag and put the correctly named residue. That should also work. The modfications actually require the hydrogen atoms to be present.

yes, when we manually change the resname in pdb from HIS to HSP, it could be recognized although not extra H proton was added in pdb file.

**Comment by pckroon** [comment]
One of the things I ran into in #631 is that there's something seriously smelly with modifications, and it tends to show for HIS-HP. For now I think the solution is to do `-mutate HIS:HSP` instead of the `modify` flag.
I *think* I know how to fix it, but it requires more mental bandwidth than I have available before my holidays. I'll have a look in February to see if I can bugsquash it.

**Comment by WangLiguo-kyrie** [comment]
okay, I see. Thanks a lot!
By the way, a small question about the modification definition of "LYS-HZ3". I'm curious why the default charged state of Lys is defined here, whereas other 3 charged residues are not.

**Comment by pckroon** [comment]
That probably has to do with the fact that modifications can only automatically detect and annotate *additional* atoms (*e.g.* protonations). Since there may be missing atoms in the input pdb file, for example because they weren't resolved in xray, missing atoms (*e.g.* deprotonations) are assumed to be missing/unresolved, rather than omitted on purpose.

**Comment by WangLiguo-kyrie** [comment]
okay, thanks for the explanation!
